### PR TITLE
chore: add default tomcat environment variables

### PR DESCRIPTION
**Pull request by fdlk**
#### Checklist
- [x] Functionality works & meets specifications
- Code reviewed
- Code unit/integration/system tested
- Migration step added in case of breaking change
- User documentation updated
- (If you have changed REST API interface) view-swagger.ftl updated
- Test plan template updated
- [x] Clean commits
- Added Feature/Fix to release notes

### CODE REVIEW

File: molgenis-app/tomcat-env [hunk 1]
```diff
@@ -0,0 +1,12 @@
+environment=production
+opencpu.uri.host=localhost
+elasticsearch.transport.addresses=localhost:9300
+db_uri=jdbc:postgresql://localhost/molgenis
+db_user=molgenis
+db_password=molgenis
+admin.password=admin
```
Comment (sidohaakma) [inline molgenis-app/tomcat-env]: There is a property file in the rpm directory with the same values. Maybe we need to make a choice how to add the properties to the deployment: https://github.com/molgenis/molgenis/blob/master/molgenis-app/rpm/templates/molgenis-server.properties. Or do we need 2?

Comment (fdlk) [reply molgenis-app/tomcat-env]: This is intended for configuration of intellij on a dev machine, so you need not manually twiddle with two different lists of env boxes. Sounds like I need a better name and/or move the file around a bit :D

Comment (fdlk) [reply molgenis-app/tomcat-env]: Better now?

Comment (sidohaakma) [reply molgenis-app/tomcat-env]: Ah right, to copy you mean! Nice.

**Pull request merged**